src/bentoml/_internal/frameworks/transformers.py

```diff
@@ -750,10 +751,29 @@ def import_model(
         resume_download=resume_download,
         **extra_hf_hub_kwargs,
     )
+
+    is_auto_class = False
```

Comment:
I don't think we should depend on this AutoModel class behaviour

Reply:
This AutoClass flag is used for when we fall back to using `config.auto_map` when `config.architectures` is unavailable. 

`config.auto_map` gives us something like `transformers.AutoModel...xyz`, where as `config.architectures` will give us something like `transformers.T5ForConditionalGeneration` (the exact model class). 

This flag is only used for local imports or snapshot download imports in the following way:
```
if is_auto_class:
    # when we don't have the exact model class, so we call from_config
    model = pretrained_model_class.from_config(
        config=config, **extra_hf_hub_kwargs
    )
else:
    # we have the exact model class, so we initalize the class using config
    model = pretrained_model_class(config=config)
```